tools/autograd/derivatives.yaml

```diff
@@ -430,7 +434,9 @@
   self: grad
   src: grad.gather(dim, index)
 
-- name: select  # TODO: ATen definition conflicts with PyTorch
+- name: select(Tensor self, int64_t dim, int64_t sliceIndex)
+  self: maybe_unsqueeze(grad, dim, self.size(dim))._unnarrow(dim, sliceIndex, self.size(dim))
```

Comment:
Is the maybe_unsqueeze call correct?  Just glancing at this, I would expect we would want to unsqueeze if we removed a dimension, which according to the docs is when the dimensionality of self > 1.  But the check passed to maybe_unsqueeze is that the size of that dimension is 1.

Reply:
Ooops. Fixed